docs/getting_started.md

```diff
@@ -19,16 +18,14 @@ plt.imshow(img[:,:,::-1])
 
 ![](../www/getting_started1.png)
 
-## Prebuilt models
+** please note that this video was made before the deepforest-pytorch -> deepforest name change. **
 
-DeepForest has a prebuilt model trained on data from 24 sites from the [National Ecological Observatory Network](https://www.neonscience.org/field-sites/field-sites-map). The prebuilt model uses a semi-supervised approach in which millions of moderate quality annotations are generated using a LiDAR unsupervised tree detection algorithm, followed by hand-annotations of RGB imagery from select sites.
+<div style="position: relative; padding-bottom: 62.5%; height: 0;"><iframe src="https://www.loom.com/embed/f80ed6e3c7bd48d4a20ae32167af3d8c" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
 
-![](../www/semi-supervised.png)
-For more details on the modeling approach see [citations](landing.html#citation).
 
-Setting the correct window size to match the prebuilt model takes a few tries. The model was trained on 0.1m data with 400px crops. For data of the same resolution, that window size is appropriate. For coarser data, we have experimentally found that larger windows are actually more useful in providing the model context (e.g 1500px windows). At some point windows become too large and the trees are too tiny to classify. Striking a balance is important.
+For single images, ```predict_image``` can read an image from memory or file and return predicted tree bounding boxes.
 
-## Sample data
+### Sample data
```

Comment:
DeepForest comes with a small set of sample data that can be used to test out the provided examples. The data resides in the DeepForest data directory. Use the ```get_data```helper function to locate the path to this directory if needed. To use images other than those in the sample data directory, provide the full path for the images.